mirrord/agent/src/steal/connection.rs

```diff
@@ -169,6 +170,7 @@ impl Client {
                     },
                     mut body,
                 ) = request.request.into_parts();
+                tracing::trace!(?connection_id, "starting request");
```

Comment:
Maybe move this to the top of this `tokio::task`? And also log `framed` and `chunked` variables